code/modules/spells/spell_types/godhand.dm

```diff
@@ -138,3 +138,28 @@
 	M.emote("scream")
 	to_chat(M, span_warning("You feel an explosion of pain erupt in your mind!"))
 	return ..()
+
+/obj/item/melee/touch_attack/raisehand
+	name = "\improper raise bloodman"
+	desc = "The skin of your palm feels thin."
+	on_use_sound = 'sound/magic/wandodeath.ogg'
+	icon_state = "flagellation"
+	item_state = "hivehand"
+	color = "#FF0000"
+/obj/item/melee/touch_attack/raisehand/afterattack(atom/target, mob/living/carbon/user, proximity)
+	var/mob/living/carbon/human/M = target
+	var/mob/living/simple_animal/hostile/asteroid/hivelord/legion/bloodman/L
```

Comment:
this temporary variable can be moved to line ~160 since you don't use it until then. Better yet, you can collapse these.
```suggestion

```